src/library_wasmfs.js

```diff
@@ -0,0 +1,6 @@
+var WASMFS = {
+    $WASMFS : {}
```

Comment:
It could be that you don't need this global object to hang stuff off.. it has proved to be an anti-pattern in some cases because it can defeat DCE.